service/service_test.go

```diff
@@ -616,3 +616,52 @@ func TestGetToken(t *testing.T) {
 	assert.Equal(t, existingIdentification.GetSecret().GetCreatedTimestamp(), secret.GetCreatedTimestamp())
 	assert.Equal(t, existingIdentification.GetSecret().GetExpirationTimestamp(), secret.GetExpirationTimestamp())
```

Comment:
missing desc

Reply:
This unit test was prior to when you told me about descriptions. I will fix them later on another branch.